bazel/repositories.bzl

```diff
@@ -44,3 +45,45 @@ py_proto_library(
 )
         """,
     )
+
+    native.new_http_archive(
+        name = "promotheus_metrics_model",
+        strip_prefix = "client_model-" + PROMETHEUS_SHA,
+        url = "https://github.com/prometheus/client_model/archive/" + PROMETHEUS_SHA + ".tar.gz",
+        build_file_content = """
+
+filegroup(
+    name = "client_model_protos_src",
+    srcs = [
+        "metrics.proto",
+    ],
+    visibility = ["//visibility:public"],
+ )
+
+proto_library(
+    name = "client_model_protos_lib",
+    srcs = [":client_model_protos_src"],
+    visibility = ["//visibility:public"],
+)
+
+cc_proto_library(
+    name = "client_model_protos",
+    deps = [":client_model_protos_lib"],
+    visibility = ["//visibility:public"],
+)
+
+py_proto_library(
```

Comment:
I would skip the Python rules unless someone is going to use them for now. Bazel really should get native py_proto_library support that builds on proto_library to make this cleaner, see https://github.com/bazelbuild/bazel/issues/2626 and https://github.com/bazelbuild/bazel/issues/3935.

Reply:
How do I skip? Initially I did not have. Then I got into some py related error. So I thought it is mandatory to have py and added this.